imports/node-app/core/util/getUserFromAuthToken.js

```diff
@@ -30,7 +31,7 @@ async function getUserFromAuthToken(loginToken, context) {
     throw new ReactionError("access-denied", "Token is not an access token");
   }
 
-  const _id = tokenObj.sub;
+  const _id = decodeAccountOpaqueId(tokenObj.sub);
```

Comment:
In a discussion with @ticean about encoding the `id` in this situation, it seems preferred to keep this as part of only GraphQL API.